#! /bin/bash

temp=$(mktemp)
trap 'rm -f "$temp"' EXIT

patch -p1 -N -r /tmp/ignore 2>&1 >"$temp" <<'EOF'
diff --git a/README.md b/README.md
index de24b35..265e6e2 100644
--- a/README.md
+++ b/README.md
@@ -14,6 +14,12 @@ Python 2.7 and 3.4+
 ## Installation & Usage
 ### pip install
 
+Just run
+
+```sh
+pip install culebratester_client --upgrade
+```
+
 If the python package is hosted on Github, you can install directly from Github
 
 ```sh
EOF

if [[ -s "$temp" ]]
then
    if grep -q 'Reversed (or previously applied) patch detected!' "$temp"
    then
        exit 0
    fi
fi
